doc/driver.md

```diff
@@ -44,31 +40,51 @@ let relativize p = Fpath.(v ".." // p)
 - `odoc/deps/stdlib/Stdlib/index.html` : Module page for the module `Stdlib`
 - `odoc/deps/astring/index.html` : astring main page
 - `odoc/deps/...` : other dependencies
+- `odoc/source/...` : rendered source files
 
-The `odoc` model for achieving this is that we have *pages* (`.mld` files) that have *children* which are either *further pages* (`.mld` files) or *modules* (from `.cmti` files). This {{!page-parent_child_spec} parent/child relationship} is specified on the command line. Parent pages must be *compiled* by `odoc` before their children. Then compiling a page `mypage.mld` will produce the file `page-mypage.odoc`.
+The `odoc` model for achieving this is that we have *pages* (`.mld` files) that have *children* which are either *further pages* (`.mld` files), *modules* (from `.cmti` files), or
+a *source parent*. This {{!page-parent_child_spec} parent/child relationship} is specified on the command line. Parent pages must be *compiled* by `odoc` before their children. Then compiling a page `mypage.mld` will produce the file `page-mypage.odoc`.
```

Comment:
```suggestion
*source trees*. This {{!page-parent_child_spec} parent/child relationship} is specified on the command line. Parent pages must be *compiled* by `odoc` before their children. Then compiling a page `mypage.mld` will produce the file `page-mypage.odoc`.
```

Reply:
Actually, I'm not sure which is best: the file is named `source-tree` but is passed as `source-parent`. Maybe we could uniformize this?

In any case, a page can have multiple source-trees as children.